pkg/network/protocols/http/ebpf_javatls.go

```diff
@@ -79,7 +79,9 @@ func (p *JavaTLSProgram) GetAllUndefinedProbes() (probeList []manager.ProbeIdent
 }
 
 func newJavaProcess(pid uint32) {
-	if err := java.InjectAgent(int(pid), javaUSMAgentJarPath, strconv.FormatInt(authID, 10)); err != nil {
+	agentArgs := make(map[string]string)
+	agentArgs["dd.usm.authID"] = strconv.FormatInt(authID, 10)
```

Comment:
add a `java_agent_cmd_args` as a new parameter under service_monitoring config section.
it is default value should be: `dd.appsec.enabled=false dd.trace.enabled=false dd.usm.enabled=true` then concat the dd.usm.authId to that string